frontend/src/metabase/components/ErrorCard.jsx

```diff
@@ -0,0 +1,42 @@
+import React, { useState, useEffect } from "react";
+import PropTypes from "prop-types";
+import BodyComponent from "metabase/components/BodyComponent";
+import Card from "metabase/components/Card";
+import Icon from "metabase/components/Icon";
+
+function ErrorCard({ errorInfo }) {
+  const [showError, setShowError] = useState(false);
+
+  useEffect(() => {
+    if (errorInfo) {
+      setShowError(true);
+    }
+  }, [errorInfo]);
+
+  return showError ? (
+    <Card className="fixed right bottom zF" p={2} m={2} width={350}>
+      <div className="flex justify-between align-center mb1">
+        <div className="text-error flex align-center">
+          <Icon name="info_outline" mr={1} size="20" />
+          <h2>Something went wrong</h2>
+        </div>
+        <Icon
+          name="close"
+          className="pl1 cursor-pointer text-dark"
+          onClick={() => setShowError(false)}
+        />
+      </div>
+      <pre style={{ height: "20vh" }} className="overflow-auto">
```

Comment:
question for @kdoh -- what's the preferred way to handle explicit heights in components like this? `styled-components`? ideally would like to avoid a `style` prop but didn't see any util classes for height.

Reply:
My highly non scientific rule of thumb is - if the item in question just needs one sort of custom thing I think a `style` prop is ok and preferable to using `styled-components` since it matches the complexity of what's actually happening. If this was the 3rd or 4th time we saw `20vh` pop up I'd push for a util class, or if we want to add more custom styling to the `pre` here I'd say do all of that and the height in a styled component.